app.js

```diff
@@ -0,0 +1,203 @@
+let books = [];
+
+function Book(title, status, author, imageUrl, numPages) {
```

Comment:
- [ ] [OPTIONAL] You've already used arrow functions in a couple of callback, why not use them more for your other functions?
It will make your code look way more modern.

- [ ] [OPTIONAL] Speaking of ES6 features, why don't you try some destructuring in order to avoid repeating 'this'?
Ex:
```
const { title, status } = this;

title = newTitle;
```
Just to practice object destructuring a bit more :-)

Reply:
Hi, thanks for your feedback it's really good, one thing to notice is that function **Book** is a constructor we can't build it by arrow functions otherwise it will lose its behavior as a constructor

Reply:
and for destructuring we are already using them in function parentheses.